src/utils/capitalize.ts

```diff
@@ -0,0 +1,3 @@
+export function capitalize(name: string) {
```

Comment:
It needs a description. Please use [JS doc](https://jsdoc.app/about-getting-started).